README.md

```diff
@@ -21,6 +21,34 @@ Moreover, with more than 20+ custom field, you can swiftly build contact forms,
 
 So weForms is an all-in-one form builder created with the global users in mind. Its minimalistic design is not only modern but also super fast and user-friendly. Everything works on a single page. Feel no delay – because your time is precious!
 
+## Development ##
+
+### Installing Dependencies ###
+Before you can use the development version of this plugin you must install the dependencies.
+
+```
+npm install
+```
+
+### Workflow ###
+Before editing or making changes check to make sure you are not editing any concatenated files. Check the Gruntfile.js in the concat: section for details.
```

Comment:
Why do I need to check this? What do I do if I'm editing a concatenated file?

Reply:
All of the concatenated files have where the source document is, but sometime it is hard to tell in sections that are larger and it may not be obvious. That is what happen to me with the when adding the weserialize function.

How about this:

Before editing or making changes check to make sure you are not editing any concatenated files. Check the Gruntfile.js in the concat: section for details. If you edit a concatenated file and run the build process any of your changes will be overwritten. 

@bwmarkle

Reply:
@avonville sounds good!